plugins/meson.build

```diff
@@ -12,6 +12,7 @@ if json_c_dep.found()
     'plugins/intel/intel-nvme.c',
     'plugins/memblaze/memblaze-nvme.c',
     'plugins/micron/micron-nvme.c',
+    'plugins/nbft/nbft-plugin.c',
```

Comment:
Could you split the plugin change into a separate commit?

Reply:
Done